TYPO3.Flow/Classes/TYPO3/Flow/Persistence/Doctrine/PersistenceManager.php

```diff
@@ -162,10 +163,14 @@ public function persistAll($onlyWhitelistedObjects = false)
 
         /** @var Connection $connection */
         $connection = $this->entityManager->getConnection();
-        if ($connection->ping() === false) {
-            $this->systemLogger->log('Reconnecting the Doctrine EntityManager to the persistence backend.', LOG_INFO);
-            $connection->close();
-            $connection->connect();
+        try {
```

Comment:
for now that seems fine to me, but yeah, I would check for a configured DB early on and maybe set a magic config value as for context?